target/linux/ramips/base-files/etc/init.d/enablemodem

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh /etc/rc.common
+. /lib/ramips.sh
+
+START=99
+
+start() {
+	local board=$(ramips_board_name)
+	if [ $board = "mr200" ]; then
+		adb shell chmod +x /WEBSERVER/www/cgi-bin/*
+		adb shell httpd -h /WEBSERVER/www/ &
+		echo "2357 000d" > /sys/bus/usb-serial/drivers/option1/new_id
```

Comment:
@heinzek
Confirmed. running the echo command after running the adb daemon, does not block the adb interface.
On a side note, this new file(enablemodem), must have +x permission set. i don't know how/where you define that in github but it ended up without +x in my build which prevented the router from running it.